streams/vn.m3u

```diff
@@ -81,10 +81,6 @@ https://livefta.malimarcdn.com/ftaedge00/pntv.sdp/playlist.m3u8
 http://27.76.137.122:1935/standard/smil:standard.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="QuangNgaiTV.vn",Quang Ngai TV (720p)
 http://118.107.85.5:1935/live/smil:PTQ.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="QuocHoiTV.vn",Quốc Hội (1080p)
-http://113.164.225.140:1935/live/quochoitvlive.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="QuocHoiTV.vn" tvg-shift="-2",Quốc Hội (720p)
-https://cloudvodqh.tek4tv.vn/live/quochoi_abr.m3u8
```

Comment:
It seems like the channel is still working:

![image](https://github.com/user-attachments/assets/b4bb0c4b-9061-451f-82e0-9836cfa581db)